tests/lib/rules/no-unused-prop-types.js

```diff
@@ -6765,7 +6848,38 @@ ruleTester.run('no-unused-prop-types', rule, {
       }, {
         message: '\'username\' PropType is defined but prop is never used'
       }]
-    }])
+    }]),
```

Comment:
the below TS tests need to be inside this call. I'll fix this when i rebase.

Reply:
OK! Thanks for your help.